ajax/libs/epub.js/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "epub.js",
+  "filename": "epub.min.js",
+  "description": "Render ePub documents in the browser, across many devices",
+  "keywords": [
+    "epub",
+    "ebooks"
+  ],
+  "author": "fchasen@gmail.com",
```

Comment:
Let's modify to `"author": "Fred Chasen <fchasen@gmail.com>"`
Because this info is better than just mention email.

Many thanks.
https://github.com/futurepress/epub.js/blob/master/bower.json#L6